query-connector/src/app/ui/designSystem/table/table.module.scss

```diff
@@ -1,62 +1,62 @@
 @use "../../../ui/styles/_variables" as *;
 
-.table {
-  display: flex;
-  flex-direction: column;
+.customizeTableWrapper {
```

Comment:
Think the table refactor may have broken the styling on some of the other pages

<img width="651" alt="Screenshot 2025-02-06 at 4 29 18 PM" src="https://github.com/user-attachments/assets/b000521c-e65e-4261-ad70-66aab4b55733" />

<img width="1053" alt="Screenshot 2025-02-06 at 4 28 25 PM" src="https://github.com/user-attachments/assets/66007d5e-3c1f-4ea3-a75c-147b8db20694" />

We also may want to extend the component to have a `borderless` variant so that we can match the stylings on the existing result page
<img width="1267" alt="Screenshot 2025-02-06 at 4 29 51 PM" src="https://github.com/user-attachments/assets/0caefbf2-f014-4c2e-8ccf-53b5c8018c29" />

Reply:
I believe I have fixed all the variations of the table in the application, if you still see any issues for sure let me know.